cognite/seismic/protos/v1/seismic_service_messages.proto

```diff
@@ -109,6 +110,7 @@ message SearchSeismicStoresRequest {
     bool include_volume_definitions = 3; // If true, includes inline/crossline volume definitions for store
     bool include_coverage = 4; // If true, includes the coverage polygon
     bool include_headers = 5; // if true, include text and binary headers in the response
+    bool include_as_geojson = 7; // If true and coverage is true, include the coverage as GeoJSON format
```

Comment:
6?

Reply:
Already in use!

Reply:
This (wkt vs geojson) would have been a good place for an enum, and maybe we should also provide an option to specify a CRS. Previous coverage APIs have allowed requesting a given CRS, which is needed for ie. Discover.

There's a CoverageParameters message in types.proto, which is _almost_ right except it uses a bool for "in_wkt" instead of an enum with the format requested. Could we improve on that and use that in the include_coverage field? (or, replace that field with a new field, more likely)